thetatauCMT/events/forms.py

```diff
@@ -1,7 +1,9 @@
 from crispy_forms.helper import FormHelper
 from crispy_forms.layout import Layout, Fieldset, Row, Submit
 from crispy_forms.bootstrap import FormActions, InlineField, StrictButton
-
+from django import forms
+from django.db import models
```

Comment:
This is nitpicking put a good chance to practice. This import does not seem to be used.
> from django.db import models
Can you see how to make a change and include in the same pull request not a new one.

Reply:
Ok I think I figured it out!